app/javascript/react/components/Map/Markers/Markers.tsx

```diff
@@ -49,8 +49,9 @@ const Markers = ({ points }: Props) => {
           position={point}
           key={point.key}
           ref={(marker) => setMarkerRef(marker, point.key)}
+          onClick={() => onMarkerClick(point.id)}
         >
-          <span className="tree">🌳</span>
+          <button className="tree">🌳</button>
```

Comment:
Hmm, real markers aren't buttons, but they are clickable. Clicking the marker centre the map and zoom in. I think it may be something we need to discuss - which approach do we prefer 🤔
<img width="626" alt="Screenshot 2024-05-31 at 10 59 17" src="https://github.com/HabitatMap/AirCasting/assets/46443841/63bbeb66-6279-4762-9fd0-9aa6ccb94d30">
U

Reply:
I will do it in a proper way after rebasing with the PR display session fixed.